gschem/src/g_hook.c

```diff
@@ -179,6 +205,7 @@ init_module_gschem_core_hook (void *unused)
   DEFINE_HOOK ("%new-page-hook",1);
   DEFINE_HOOK ("%action-property-hook",3);
   DEFINE_HOOK ("%bind-keys-hook",3);
+  DEFINE_HOOK ("%switch-action-mode",1);
```

Comment:
Maybe the hook name should end with `-hook`, for consistency with other hook variables.